extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetToolsTestDataTest.java

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.table;
+
+import io.deephaven.engine.exceptions.TableInitializationException;
+import io.deephaven.engine.table.Table;
+import io.deephaven.engine.table.impl.locations.TableDataException;
+import io.deephaven.engine.testutil.junit4.EngineCleanup;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.nio.file.Path;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.failBecauseExceptionWasNotThrown;
+import static org.junit.Assume.assumeTrue;
+
+/**
+ * Assumes that there is already a checkout of
+ * <a href="https://github.com/deephaven/deephaven-core-test-data">deephaven-core-test-data</a>. This is currently meant
```

Comment:
I had no idea this data existed. I am glad these tests are not failing after all the changes we have been making.

Re-ran these on my machine too and all passed.

On a side note (and not necessarily your responsibility for this PR), I know there is also test data in https://github.com/deephaven/examples for parquet. I usually test against that. This is my test script that I usually use

```
from deephaven import parquet

a = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/Pems/parquet/pems/").select()

b = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/CryptoCurrencyHistory/Parquet/crypto_sept8.parquet").select()

c = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/CryptoCurrencyHistory/Parquet/CryptoTrades_20210922.parquet").select()

d = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/CryptoCurrencyHistory/Parquet/crypto.parquet").select()

e = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/CryptoCurrencyHistory/Parquet/crypto_sept7.parquet").select()

f = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/CryptoCurrencyHistory/Parquet/FakeCryptoTrades_20230209.parquet").select()

g = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/Taxi/parquet/taxi.parquet").select()

h = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/SensorData/parquet/SensorData_gzip.parquet").select()

i = parquet.read("/Users/shivammalhotra/deephaven/projects/deephaven-examples/Taxi/parquet").select()
```

Do you think we should add similar tests like this file for data in deephaven-examples?

Reply:
I think adding a similar test to what I've done here would be reasonable. Long term we would like it to be more automated, but happy to take smaller manual steps towards that goal.

Reply:
I can do this part then.